presto-vertica/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.prestosql</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>348-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-vertica</artifactId>
+    <description>Presto - Vertica Connector</description>
+    <packaging>presto-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.prestosql</groupId>
+            <artifactId>presto-base-jdbc</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>configuration</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.vertica.jdbc</groupId>
+            <artifactId>vertica-jdbc</artifactId>
```

Comment:
On https://search.maven.org/artifact/com.vertica.jdbc/vertica-jdbc/10.0.1-0/jar i cannot find license information (i tried to find it in pom.xml  and jar). Also https://www.vertica.com/download/vertica/client-drivers/ doesn't contain "license" information explicitly. What license is this driver available on?

Reply:
The license exists here, it's no problem in my understanding. 
https://www.vertica.com/wp-content/uploads/2020/10/2020-10-01-Final-Vertica-Client-License-Agreement.pdf
> Vertica provides downloads of clients and drivers that are redistributable. Open source clients and
drivers are governed by their applicable open source license terms

Reply:
@martint do you want to check the Apache license compatibility?

Reply:
ping @martint

Reply:
I'm asking to Vertica support team just in case.

Reply:
@martint @findepi I received an answer from Vertica support (in Japanese). They said it's no problem unless we violate `Distribution Restrictions` in the PDF. 

> You may not alter any copyright, trademark or patent notice in the Distributable Code; include
Distributable Code in malicious, deceptive or unlawful programs; or modify or distribute the source code
of any Distributable Code

The last sentence `or modify or distribute the source code of any Distributable Code` is unrelated to JDBC driver because the source code isn't published.

Reply:
Support's answer is not necessarily legally binding (it's not the license itself), but if I am understanding correctly both what they wrote & Apache License requirements, the exclusion of "deceptive programs" would not be compatible with AL. Our program is not "deceptive", but our  license doesn't forbid building "deceptive" programs as downstream projects.
IANAL though.

@martint ping

Reply:
Maybe we could move forward without a driver,  by using scope provided for dependency, so users would need to bring their own Vertica driver. Then once we solve the issue and it is not a problem then we could add driver (changing the scope to compile).

Reply:
Thanks your comments. Let me apply the suggestion just before merge.

Reply:
This language in the license is problematic and potentially incompatible with the Apache license:

> you must display your valid copyright notice on your programs; 

> and indemnify, defend, and hold harmless Micro Focus from any claims, including attorneys’ fees, related to the distribution or use of your programs

Reply:
License is updated with the new release and good to go.

https://central.sonatype.com/artifact/com.vertica.jdbc/vertica-jdbc/24.1.0-2/overview

https://www.vertica.com/wp-content/uploads/2024/02/Vertica-Client-License-Agreement_Feb2023.pdf